worker/src/RTC/SeqManager.cpp

```diff
@@ -77,28 +77,30 @@ namespace RTC
 		if (!this->dropped.empty())
 		{
 			// Delete dropped inputs older than input - MaxValue/2.
-			size_t droppedSize = this->dropped.size();
-			auto it            = this->dropped.lower_bound(input - MaxValue / 2);
+			size_t dropped = this->dropped.size();
```

Comment:
Can you rename this `size_t dropped` variable to `size_t droppedCount`?

Reply:
yep